Packs/JoeSecurity/Integrations/JoeSecurity/JoeSecurity.py

```diff
@@ -102,7 +102,7 @@ def analysis_to_entry(title, info):
 
         dbot_scores.append({
             'Vendor': 'JoeSecurity',
-            'Indicator': analysis_info['MD5'],
+            'Indicator': analysis_info['MD5'] if analysis_info['MD5'] else analysis['filename'],
```

Comment:
The bug was that `analysis_info` did not have an `MD5` key? if this is the case we will get an exception here too.